### PR TITLE
LPS-183521 | Fix to solve Ci problem of not deleting Data Set Views

### PR DESCRIPTION
**Pull request by abhnerramos**
**Description**
Fixes to solve Ci problem of not deleting Data Set Views
- adding deleteAllDatasetEntries macro on setUp to solve the CI's problem of starting with some dataset entries already created
- changing the name "Test Dataset" to 'Dataset Test' because on setup the dataset name is 'Dataset Test'

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-183521

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:stop

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase [hunk 1]
```diff
@@ -16,6 +16,12 @@ definition {
 		task ("Given creating a Data Set via API") {
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase:16]:
```suggestion
		task ("Given the list of dataset entries is empty ") {
                        if (IsElementPresent(key_itemName = "", locator1 = "FrontendDataSetAdmin#TABLE_CELL")) {
				FrontendDataSetAdmin.deleteAllDatasetEntries();
			}
		}
	task ("Given creating a Data Set via API") {
	...
```

**Review by mateusralv — CHANGES_REQUESTED** [review]

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
These pushes are just to fix some accidental pushes that I did to this branch.

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
@mateusralv , can you check this, please? Are there any changes to be made, or can we forward it?

**Review by mateusralv — APPROVED** [review]
GREAT!!!!! 🔝 🚀

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**